In [1]:
import pandas as pd 
pd.options.display.max_rows = 400

In [2]:
moral_words = pd.read_pickle('../data/moral_words.pkl')
moral_words['category'] = moral_words.foundation
moral_words = moral_words['category']

neutral_words = pd.read_pickle('../data/neutral_words.pkl')
neutral_words['category'] = 'neutral'
neutral_words = neutral_words['category']

non_words = pd.read_pickle('../data/nonwords.pkl')
non_words['category'] = 'nonword'
non_words = non_words['category']

In [3]:
all_words = pd.concat([moral_words,neutral_words,non_words])

In [4]:
all_words.value_counts()

nonword             170
neutral              50
care.vice            12
authority.virtue     12
sanctity.vice        12
care.virtue          12
sanctity.virtue      12
fairness.virtue      12
loyalty.virtue       12
authority.vice       12
loyalty.vice         12
fairness.vice        12
Name: category, dtype: int64

## Words Per Task

Lexical Decision Task:
- 10 per foundation (5 virtue and 5 vice per foundation) = 50 moral words
- 50 Neutral words
- 100 Non-words

Affect Misattribution:
- 14 per foundation = 70 moral words
- 70 non-words

Total:
- 120 moral words
- 50 neutral words
- 170 nonwords

In [5]:
stimuli = pd.DataFrame(all_words)

In [6]:
stimuli['correct_amp'] = 'nan'
stimuli['correct_ldt'] = 'nan'

In [7]:
for i,row in stimuli.iterrows():
    cat = row.category
    if cat.endswith('.vice'):
        stimuli.at[i, 'correct_amp'] = 'r'
        stimuli.at[i, 'correct_ldt'] = 'l'
    elif cat.endswith('.virtue'):
        stimuli.at[i, 'correct_amp'] = 'l' 
        stimuli.at[i, 'correct_ldt'] = 'l'
    elif cat == 'nonword':
        stimuli.at[i, 'correct_ldt'] = 'r'
    elif cat == 'neutral':
        stimuli.at[i, 'correct_ldt'] = 'l'

In [8]:
stimuli['word'] = stimuli.index
stimuli = stimuli.reset_index()
del stimuli['index']

In [9]:
stimuli[['word','category','correct_amp','correct_ldt']]

word          category correct_amp correct_ldt
0             care       care.virtue           l           l
1             help       care.virtue           l           l
2           health       care.virtue           l           l
3             good       care.virtue           l           l
4            peace       care.virtue           l           l
5             safe       care.virtue           l           l
6       compassion       care.virtue           l           l
7           safety       care.virtue           l           l
8          improve       care.virtue           l           l
9           growth       care.virtue           l           l
10        increase       care.virtue           l           l
11         restore       care.virtue           l           l
12          killed         care.vice           r           l
13           death         care.vice           r           l
14        violence         care.vice           r           l
15         attacks         care.vice           r           l
16       civilians         care.vice           r           l
17         suicide         care.vice           r           l
18            died         care.vice           r           l
19         torture         care.vice           r           l
20         victims         care.vice           r           l
21          victim         care.vice           r           l
22         injured         care.vice           r           l
23            shot         care.vice           r           l
24            free   fairness.virtue           l           l
25          ensure   fairness.virtue           l           l
26         protect   fairness.virtue           l           l
27         benefit   fairness.virtue           l           l
28       integrity   fairness.virtue           l           l
29       interests   fairness.virtue           l           l
30          credit   fairness.virtue           l           l
31       agreement   fairness.virtue           l           l
32     opportunity   fairness.virtue           l           l
33            fair   fairness.virtue           l           l
34             win   fairness.virtue           l           l
35        promises   fairness.virtue           l           l
36           fraud     fairness.vice           r           l
37         illegal     fairness.vice           r           l
38           crime     fairness.vice           r           l
39            hate     fairness.vice           r           l
40          prison     fairness.vice           r           l
41         accused     fairness.vice           r           l
42            lies     fairness.vice           r           l
43         charges     fairness.vice           r           l
44      threatened     fairness.vice           r           l
45           voter     fairness.vice           r           l
46          forced     fairness.vice           r           l
47      terrorists     fairness.vice           r           l
48         support    loyalty.virtue           l           l
49            love    loyalty.virtue           l           l
50    intelligence    loyalty.virtue           l           l
51          strong    loyalty.virtue           l           l
52         friends    loyalty.virtue           l           l
53           party    loyalty.virtue           l           l
54           great    loyalty.virtue           l           l
55         loyalty    loyalty.virtue           l           l
56            hope    loyalty.virtue           l           l
57        promised    loyalty.virtue           l           l
58         praised    loyalty.virtue           l           l
59         sharing    loyalty.virtue           l           l
60       terrorist      loyalty.vice           r           l
61             ban      loyalty.vice           r           l
62          groups      loyalty.vice           r           l
63         failure      loyalty.vice           r           l
64           black      loyalty.vice           r 

In [10]:
stimuli[['word','category','correct_amp','correct_ldt']].to_csv('words4exp.csv')